KinSDK/KinSDK/Core/KinAccount.swift

```diff
@@ -276,21 +167,42 @@ final class KinStellarAccount: KinAccount {
         }
     }
 
-    func generateTransaction(to recipient: String, kin: Kin, memo: String? = nil, fee: Stroop) -> Promise<TransactionEnvelope> {
+    /**
+     Build a Kin transaction for a specific address.
+
+     - Parameter recipient: The recipient's public address.
+     - Parameter kin: The amount of Kin to be sent.
+     - Parameter memo: An optional string, up-to 28 bytes in length, included on the transaction record.
+     - Parameter fee: The fee in `Quark`s used if the transaction is not whitelisted.
+
+     - Returns: A promise which is signalled with the `Transaction.Envelope` or an `Error`.
+     */
+    public func buildPaymentTransaction(to recipient: String, kin: Kin, memo: String? = nil, fee: Quark) -> Promise<Transaction.Envelope> {
```

Comment:
I'm missing a general TransactionBuilder method `getTransactionBuilder` that will be the entry point for building complex transactions.